Lib/pydoc.py

```diff
@@ -390,9 +391,7 @@ def fail(self, object, name=None, *args):
 
     docmodule = docclass = docroutine = docother = docproperty = docdata = fail
 
-    def getdocloc(self, object,
-                  basedir=os.path.join(sys.base_exec_prefix, "lib",
-                                       "python%d.%d" %  sys.version_info[:2])):
+    def getdocloc(self, object, basedir=sysconfig.get_path('stdlib')):
```

Comment:
This looks like a good fix regardless of the lib/lib64/debian-triplet question!

Maybe open a distinct ticket and cherry-pick this change in a PR?

Reply:
There was already https://bugs.python.org/issue21016 issue for that and I proposed a separted PR: https://github.com/python/cpython/pull/18476